charts/multicluster-controlplane/templates/secret.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: controlplane-config
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+{{- with .Values.ocmconfig }}
+stringData:
```

Comment:
can we only use `stringData`? 

```
stringData:
  ocmconfig.yaml: |-
      dataDirectory: {{ .dataDirectory }}
      apiserver:
            externalHostname: {{ .apiserver.externalHostname }}
            port: {{ .apiserver.port}}
            {{- if and .apiserver.caFile .apiserver.caKeyFile }}
            caFile: "apiserver-ca.crt"
            caKeyFile: "apiserver-ca.key"
            {{- end }}
      etcd:
            mode: {{ .etcd.mode }}
            prefix: {{ .etcd.prefix }}
            {{- if (and (eq .etcd.mode "external") (and .etcd.caFile (and .etcd.certFile (and .etcd.keyFile .etcd.servers)))) }}
            servers: {{ .etcd.servers }}
            caFile: etcd-ca.crt
            certFile: etcd-cert.crt
            keyFile: etcd-cert.key
            {{- end }}
  {{- if and .apiserver.caFile .apiserver.caKeyFile }}
  apiserver-ca.crt: |-
        {{ .Files.Get .apiserver.caFile }}
  apiserver-ca.key: |-
        {{ .Files.Get .apiserver.caKeyFile }}
  {{- end }}
  {{- if (and (eq .etcd.mode "external") (and .etcd.caFile (and .etcd.certFile (and .etcd.keyFile .etcd.servers)))) }}
  etcd-ca.crt: |-
        {{ .Files.Get .etcd.caFile }} 
  etcd-cert.crt: |-
        {{ .Files.Get .etcd.certFile  }}
  etcd-cert.key: |-
        {{ .Files.Get .etcd.keyFile  }}
  {{- end}}
```

we need test using ca files in future, and may update the code to support this